pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -86,7 +86,7 @@ func (a Adapter) Push(parameters common.PushParameters) (err error) {
 	}
 
 	// Wait for Pod to be in running state otherwise we can't sync data or exec commands to it.
-	pod, err := a.waitAndGetComponentPod(false)
+	pod, err := a.waitAndGetComponentPod(true)
```

Comment:
We set `hideSpinner` to true here, that means we don't need to spinner for waiting pods?

Reply:
The spinner is not necessary here, we're just using that to retrieve the pod. The actually waiting for the component to start is in the Rollout function.